### ISSUE TITLE
Modern art like if rewritings (missing code, wrong condition merging)

### ISSUE THREAD

**Issue by cal101** [walkmod/walkmod-pmd-plugin#5]
```
         try {
             pixelboxx = adminServlet.getCheckedPixelbox();
             action = callCtx.getInt("action", 0);
-            if (action == 0) {
-                if (callCtx.getString("startjob", null) != null) {
-                    action = 1;
-                }
+            if (action == (callCtx.getString("startjob", null) != null)(action == 0) && (callCtx.getString("startjob", null) != null)) {
             }

```

**Comment by rpau** [comment]
Thanks, this part probably is when walkmod rewrites the minimum set of changes, but thanks a lot

**Comment by rpau** [comment]
Fixed at 4617abc2df601b7115a35beedc1aeaffaf79586c
